.evergreen/config.yml

```diff
@@ -2777,6 +2834,12 @@ buildvariants:
     tasks:
       - test-aws-lambda-task-group
 
+  - matrix_name: "searchindex-test"
+    matrix_spec: { version: ["7.0"], os-faas-80: ["rhel80-large-go-1-20"] }
+    display_name: "Search Index ${version} ${os-faas-80}"
+    tasks:
+      - test-search-index-task-group
```

Comment:
Is there a reason the search index test needs its own matrix and task group? Does it start up or tear down special resources?

Reply:
Yes, we need to start an atlas cluster to connect the server there.